### PR TITLE
Text Toto

### PR DESCRIPTION
**Pull request by SirSamTumless**
That didn't go as planned....

### CODE REVIEW

File: A.py [hunk 1]
```diff
@@ -1 +1,33 @@
-print("Hey let's print")
+print("sssssoo++:          -//+++++++oyhddmmmmmmmNNNNNNNNNNNNNNNNNNNNmmmhyo/:---------------.....``..````..
+sssssoo++/.         .://+++++++yhdmmmdhhdmmddddmddmNNmmmNNNmmmmddhhs+:-::-------------......````..:/
+sosssooo+/:         `://///++shmmNmdsoydddhhhhhhhhmmddddmmmmdddhddyys/:::::::::::::::----..``...:+/+
+ssssssoo++/.         -:////+sydmNNh++oyyhsyyyyyyyyddhdhhhdmdhyyhddhs+:-::::::::::::::::-......:++/:.
+ssssssooo+/:`        .://///osdNNh///osyoosssssssyhhyyyyyyhhhsoshhdh+--:::::::::::::::-....-/++/-...
+sssssssoo++/-        `-://///+dNd+///++o++oooossssyysssssyyyys+++ohms--:::-:::::::::-....-/o+/-.....
+ssssoossoo++/.        -://////dmy////+++++++ooooooosoooooooso+////sdh+::-::::::::::----:+oo/-......`
+sssssosssoo++:`       .://////ymo///+ooosssssooooooooooooooooo///:+mdo--::::::::-----:oss/-.......:o
+ssssssosoooo+/-       .://////oms/ooyyhdmmmmdhyssssyyhddmddhyyo//-+myo:::::::::----/oss/-...-:/:+yhs
```
Comment (tbille) [inline A.py]: add ticket or die

Comment (charlesjquinn) [reply A.py]: OMG.

**Comment by anasereijo** [comment]
horrible code 👎

File: A.py (hunk 1 above)
Comment (tbille) [reply A.py]: :fire:

Comment (tbille) [reply A.py]: P45

**Pull request closed without merging**